Spontaneous test failure: Multi-Z debug can't find a blobstart landmark for stationloving component

[8_Run Tests.txt](https://github.com/tgstation/tgstation/files/8905452/8_Run.Tests.txt)

https://github.com/tgstation/tgstation/actions/runs/2499436037/attempts/1

```
2022-06-15T03:03:38.8328992Z ##[error][02:58:06] Runtime in stationloving.dm,41: Unable to find a blobstart landmark
  proc name: relocate (/datum/component/stationloving/proc/relocate)
  src: /datum/component/stationloving (/datum/component/stationloving)
  call stack:
  /datum/component/stationloving (/datum/component/stationloving): relocate()
  /datum/component/stationloving (/datum/component/stationloving): Initialize(0, 0)
  /datum/component/stationloving (/datum/component/stationloving): New(/list (/list))
  the cheap plastic imitation of... (/obj/item/disk/nuclear/fake/obvious):  AddComponent(/list (/list))
  the cheap plastic imitation of... (/obj/item/disk/nuclear/fake/obvious): Initialize(0)
  Atoms (/datum/controller/subsystem/atoms): InitAtom(the cheap plastic imitation of... (/obj/item/disk/nuclear/fake/obvious), 0, /list (/list))
  the cheap plastic imitation of... (/obj/item/disk/nuclear/fake/obvious): New(0)
  /datum/unit_test/create_and_de... (/datum/unit_test/create_and_destroy): Run()
  RunUnitTest(/datum/unit_test/create_and_de... (/datum/unit_test/create_and_destroy), /list (/list))
  RunUnitTests()
  /datum/callback (/datum/callback): InvokeAsync()
```

--- a/_maps/map_files/debug/multiz.dmm
+++ b/_maps/map_files/debug/multiz.dmm
@@ -1360,6 +1360,10 @@
 /obj/effect/spawner/structure/window/reinforced,
 /turf/open/floor/plating,
 /area/station/engineering/storage)
+"DN" = (
+/obj/effect/landmark/blobstart,
+/turf/open/floor/iron,
+/area/station/hallway/primary/central)
 "DU" = (
 /obj/structure/table,
 /obj/item/flashlight{
@@ -3009,7 +3013,7 @@ eb
 eb
 bu
 cD
-bE
+DN
 bE
 Rl
 Ra

